doc/api/http.markdown

```diff
@@ -64,10 +64,10 @@ a `'close'` event or a special `'agentRemove'` event. This means that if
 you intend to keep one HTTP request open for a long time and don't
 want it to stay in the pool you can do something along the lines of:
 
-    http.get(options, function(res) {
+    http.get(options, (res)=> {
```

Comment:
agreed, I'm only seeing `(args) => {` in use in the wild at the moment with the extra space (not that I'm looking very widely)